src/components/Dropdown/index.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from "react";
```

Comment:
Please add `// @flow` for static typing and fix typing when other PR comments will be resolved.